src/components/LocationEditor/index.tsx

```diff
@@ -18,25 +18,44 @@ const initialerrorMessages: ErrorMessages = {
 
 interface LocationEditorProps {
 	location?: Location;
-	submitHandler: (e: FormEvent<HTMLFormElement>, newLocation: CreateLocation | Location) => void;
+	submitHandler: (
+		e: FormEvent<HTMLFormElement>,
+		newLocation: CreateLocationRequest | Location
+	) => void;
 	submitLabel: string;
 }
 
+const boroughs: Array<CreateLocationRequest['borough']> = [
+	'Mitte',
+	'Friedrichshain-Kreuzberg',
+	'Pankow',
+	'Charlottenburg-Wilmersdorf',
+	'Spandau',
+	'Steglitz-Zehlendorf',
+	'Tempelhof-Schöneberg',
+	'Neukölln',
+	'Treptow-Köpenick',
+	'Marzahn-Hellersdorf',
+	'Lichtenberg',
+	'Reinickendorf',
+	'außerhalb',
+];
+
 const LocationEditor: FC<LocationEditorProps> = ({ location, submitHandler, submitLabel }) => {
-	const [locationObject, locationObjectSet] = useState<CreateLocation | Location | undefined>(
-		location || undefined
-	);
+	const [locationObject, locationObjectSet] = useState<
+		CreateLocationRequest | Location | undefined
+	>(location || undefined);
 	const [errorMessages, errorMessagesSet] = React.useState<ErrorMessages>(initialerrorMessages);
 	const [formValid, formValidSet] = useState<boolean>(true);
 
-	const barrios = Object.entries(CreateLocation.borough).map(([key, value]) => ({
-		value: key,
-		label: value,
+	const barrios = boroughs.map((borough) => ({
```

Comment:
Here other variable names would make more sense. I think boroughs should rather be berlinDistricts and barrios could just be locationDistrics or sth like that

Reply:
Yep! I went with `const districtOptions = berlinDistricts.map((berlinDistrict) => ...` now.